apps/dashboard/src/components/product-pages/common/Topnav.tsx

```diff
@@ -1,43 +1,20 @@
-import { Box, Container, Flex, useBreakpointValue } from "@chakra-ui/react";
-import { useScrollPosition } from "@n8tb1t/use-scroll-position";
+import { Box, Container, Flex } from "@chakra-ui/react";
 import { Logo } from "components/logo";
-import { useState } from "react";
 import { TrackedLink } from "tw-components";
 import { DesktopMenu } from "./nav/DesktopMenu";
 import { MobileMenu } from "./nav/MobileMenu";
 
 export const HomepageTopNav: React.FC = () => {
-  const isMobile = useBreakpointValue({ base: true, md: false });
-  const [isScrolled, setIsScrolled] = useState(false);
-
-  useScrollPosition(
-    ({ currPos }) => {
-      if (currPos.y < -5) {
-        setIsScrolled(true);
-      } else if (currPos.y >= -5) {
-        setIsScrolled(false);
-      }
-    },
-    [isMobile],
-    undefined,
-    false,
-    16,
-  );
-
   return (
     <Box
-      transition="all 50ms"
       position="sticky"
       top={0}
       left={0}
       w="100%"
-      willChange="backdrop-filter background-color"
       zIndex="overlay"
       as="header"
-      bgColor={isScrolled ? "rgba(0,0,0,0.8)" : "rgba(0,0,0,0)"}
-      backdropFilter={
-        isScrolled ? "saturate(180%) blur(20px)" : "saturate(100%) blur(0px)"
-      }
+      bgColor="rgba(0,0,0,0)"
```

Comment:
this is not looking good when you scroll the page down

![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/yNOf1svJ8o3zjO7zQouZ/1299db78-1c55-4b2d-8818-5b9c2e7abf64.png)

Reply:
ah

Reply:
@MananTank I gave it a default blur-20px. 

![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/VtULqGKSp33n0VHOzNJK/b11a1822-9f93-483d-a5a4-0f3c715e631f.png)

This page is very soon to be deleted by so it's up to you. I'm fine with closing this PR

https://www.notion.so/thirdweb/Pages-that-should-move-into-framer-or-be-removed-12e8141725338051a965daa7ae2e60dc

Reply:
yeah let's not do this - also let's not waste time on pages that will be deleted or moved to framer soon, we have lots to do on "in app" pages :)

Reply:
Understood ^